src/Screens/utils.tsx

```diff
@@ -0,0 +1,21 @@
+import {
+  getFocusedRouteNameFromRoute,
+  RouteProp,
+} from "@react-navigation/native";
+import { RootStackParamList } from "./types";
+
+export const getHeaderTitle = (
+  route: RouteProp<RootStackParamList, "PokemonListTab">
+) => {
+  // If the focused route is not found, we need to assume it's the initial screen
+  // This can happen during if there hasn't been any navigation inside the screen
+  // In our case, it's "Search" as that's the first screen inside the navigator
+  const routeName = getFocusedRouteNameFromRoute(route) ?? "Search";
+
+  switch (routeName) {
+    case "Search":
+      return "Search";
+    case "Liked":
+      return "Liked";
+  }
```

Comment:
What does this code do? 🤔

Reply:
![Screenshot 2024-09-02 at 14 28 34](https://github.com/user-attachments/assets/af883b5b-b4e1-43de-8fd6-2d3462c89605)
It set the proper header title in the navigation, in this case is "Liked"

Reply:
This approach can be used anytime you want to set options for a parent navigator based on a child navigator's state

Reply:
The alternative is to use a Stack navigator inside each tab

Reply:
I mean, for now, can it be just
```suggestion
  return routeName;
```
?